src/layers/fp64/scatterplot-layer/scatterplot-layer-vertex.glsl

```diff
@@ -22,47 +22,49 @@
 
 attribute vec3 positions;
 attribute vec4 instancePositionsFP64;
-attribute vec2 layerHeight;
+attribute vec2 instanceHeightsFP64;
 attribute float instanceRadius;
 attribute vec4 instanceColors;
 attribute vec3 instancePickingColors;
 
 // Only one-dimensional arrays may be declared in GLSL ES 1.0. specs p.24
 uniform float opacity;
-uniform vec2 zoomRadiusFP64;
+uniform float radius;
 uniform float radiusMinPixels;
 uniform float radiusMaxPixels;
 uniform float renderPickingBuffer;
 
 varying vec4 vColor;
 
 void main(void) {
-  // For some reason, need to add one to elevation to show up in untilted mode
+  // Multiply out radius and clamp to limits
+  float radiusPixels = clamp(
+    project_scale(radius * instanceRadius),
+    radiusMinPixels, radiusMaxPixels
+  );
 
   vec2 projected_coord_xy[2];
   project_position_fp64(instancePositionsFP64, projected_coord_xy);
 
-  vec2 pos_mul_radius[4];
-  vec4_fp64(vec4(positions * instanceRadius, 0.0), pos_mul_radius);
-
   vec2 vertex_pos_localspace[4];
-  vec4_scalar_mul_fp64(pos_mul_radius, zoomRadiusFP64, vertex_pos_localspace);
+  vec4_fp64(vec4(positions * radiusPixels, 0.0), vertex_pos_localspace);
 
-  vec2 vertex_pos_modelspace[4];
+  // vec2 pos_mul_radius[4];
+  // vec4_fp64(vec4(positions * radiusPixels, 0.0), pos_mul_radius);
+
+  // vec2 vertex_pos_localspace[4];
+  // vec4_scalar_mul_fp64(pos_mul_radius, radius, vertex_pos_localspace);
 
+  vec2 vertex_pos_modelspace[4];
```

Comment:
> Also, ^ is this + 1.0 corresponding to the comment need to add one to elevation to show up in untilted mode? if I recall it was due to the near clipping plane was set to 0.1, so zero z=0 will be clipped and we should have fixed for that.

@gnavvy Shaojing increased the far clipping plane a while ago, so we no longer need to add a fudge factor to the z coordinates.